src/lib/balance-providers/utils.ts

```diff
@@ -0,0 +1,16 @@
+import type {ChainConfig} from '~/config'
+
+import {BalanceProviders} from './types'
+import {BloksProvider} from './bloks'
+import {LightAPIProvider} from './light-api'
+
+export function getBalanceProvider(name: BalanceProviders, chain: ChainConfig) {
+    switch (name) {
+        case BalanceProviders.Bloks:
+            return new BloksProvider(chain)
+        case BalanceProviders.LightAPI:
+            return new LightAPIProvider(chain)
+        default:
+            return new LightAPIProvider(chain)
```

Comment:
Shouldn't this throw or return null if no provider is available, won't the LightAPIProvider error on any but supported chain?